compose_test.go

```diff
@@ -90,68 +89,91 @@ func ExampleLocalDockerCompose_WithEnv() {
 	})
 }
 
+func enumDockerComposes(filePaths []string, identifier string, executor func(compose *LocalDockerCompose), t *testing.T) {
+	composes := []*LocalDockerCompose{
+		NewLocalDockerCompose(filePaths, identifier, WithLogger(TestLogger(t))),
+		NewContainerizedDockerCompose(filePaths, identifier, WithLogger(TestLogger(t))),
```

Comment:
I think we are using the new Dockerised compose here. Could you update the docs with examples on how to use it? There is a dedicated section for compose in the docs/features dir.

Reply:
Done